src/checks/blockbreak/InstaBreak.php

```diff
@@ -71,6 +71,13 @@ public function checkEvent(Event $event, PlayerAPI $playerAPI) : void {
 					return;
 				}
 				$target = $event->getBlock();
+
+				// pocketmine seems to not compensate for bamboo break time
+				// we'll ignore this so long.
+				if ($target->getTypeId() == BlockTypeIds::BAMBOO) {
+                                         return;
+				}
+				
```

Comment:
This does not completely fix the bug. Many other blocks have false positives, and Bamboo block is one of them.